Glynet/dist/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+   "author": {
+      "name": "Arda",
+      "id": "403225819222245377"
+   },
+   "url": [
+      "www.glynet.com",
+      "glynet.com",
+      "https://glynet.com"
+   ],
+   "description": {
+      "tr": "Glynet, yeni açılmış bir sosyal medya sitesi. Küçük bir müzik kütüphanesi de bulunmakta.",
+      "en": "Glynet is a social media website that launched newly. Includes a music library as well."
+   },
+   "service": "Glynet",
+   "version": "1.0.0",
+   "logo": "https://i.hizliresim.com/kM5DdD.png",
+   "thumbnail": "https://i.hizliresim.com/WXjvJ2.png",
```

Comment:
My only complaint about the metadata is that the thumbnail includes taskbar icons and your mouse cursor. It would be great if you could remove that.